saleor/graphql/payment/tests/mutations/test_transaction_initialize.py

```diff
@@ -234,6 +236,39 @@ def test_for_checkout_without_payment_gateway_data(
     assert checkout.authorize_status == CheckoutAuthorizeStatus.PARTIAL
 
 
+def test_for_checkout_transactions_limit(user_api_client, checkout_with_prices):
+    # given
+    TransactionItem.objects.bulk_create(
+        [
+            TransactionItem(
+                checkout=checkout_with_prices, currency=checkout_with_prices.currency
+            )
+            for _ in range(TransactionSessionBase.TRANSACTION_ITEMS_LIMIT)
```

Comment:
Why you didn't mock the TRANSACTION_ITEMS_LIMIT to reduce a limit? This test will be slow, as you're creating the 100 Transactions. The test will have the same result when you change the limit to 2 transactions instead of 100.

Reply:
When we move this variable to settings we could use `@override_settings` instead of mock.